scripts/blog_template.py

```diff
@@ -40,27 +74,48 @@ def fix_img_paths(data: pd.DataFrame) -> pd.Series:
     A function for fixing image paths generated by Microsoft forms
     """
 
-    assert 'image_file' in data.columns 
+    assert "image_file" in data.columns
+
+    return (
+        data["image_file"]
+        .fillna("")
+        .apply(
+            lambda x: ",".join(
+                [x.split("/")[-1].replace("%20", "_") for x in x.split(";")]
+            )
+        )
+    )
 
-    return data['image_file'].fillna("").apply(lambda x: x.split("/")[-1].replace("%20","_"))
 
-def main(data_file, output_path):
+def main(data_file: str, output_path: str, date: str) -> None:
     """
     Main script for loading data in pandas iterating over rows and passing it to render_md
     """
     # open data file from csv
     # expects csv file containing windows encoding from forms
-    working_file = pd.read_csv(data_file)
+    working_file = pd.read_csv(data_file, encoding="latin")
 
-    working_file['image_file'] = fix_img_paths(working_file)
+    working_file = column_mapper(working_file)
+
+    working_file["image_file"] = fix_img_paths(working_file)
+
+    # randomly shuffle the rows
+    working_file = working_file.sample(frac=1, random_state=42).reset_index(drop=True)
```

Comment:
sample randomly samples our dataframe. Frac=1 just returns the whole thing, `random_state` just means the randomisation is deterministic for reruns of the script on your machine.